code/index.html

```diff
@@ -8,26 +8,61 @@
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
       rel="stylesheet" />
+      <!--Big title-->
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Monoton&display=swap" rel="stylesheet">
+      <!--sound on title-->
+      <link rel="preconnect" href="https://fonts.googleapis.com">
+      <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+      <link href="https://fonts.googleapis.com/css2?family=Covered+By+Your+Grace&display=swap" rel="stylesheet">
     <title>Chat Bot</title>
   </head>
-
   <body>
-    <h1>Welcome to xxx.</h1>
-    <h2>Ask about xxxx.</h2>
-    <main>
-      <section class="chat" id="chat"></section>
+    <!--CHAT BAR BLOCK-->
+    <div class="overlay">
+      <div class="overlay-inside">
+    <h1>Welcome to The Beat Bot!</h1>
+    <p class="signature">Designed and Developed by Madeleine Svensson & Lovisa Carling</p>
```

Comment:
Dont know why I haven't put my name in my projects so far, will definitely do it as well end be proud of the result :D